### PR TITLE
Made use of map[string]string for nw options in default-gateway impl

### PR DESCRIPTION
**Pull request by mavenugo**
Signed-off-by: Madhu Venugopal <madhu@docker.com>

### CODE REVIEW

File: default_gateway_freebsd.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package libnetwork
+
+import "github.com/docker/libnetwork/types"
+
+func (c *controller) createGWNetwork() (Network, error) {
+	return nil, types.NotImplementedErrorf("Default Gateway functionality is not implemented in freebsd")
```
Comment (mrjana) [inline default_gateway_freebsd.go]: error string to lower case

File: default_gateway_windows.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package libnetwork
+
+import "github.com/docker/libnetwork/types"
+
+func (c *controller) createGWNetwork() (Network, error) {
+	return nil, types.NotImplementedErrorf("Default Gateway functionality is not implemented in windows")
```
Comment (mrjana) [inline default_gateway_windows.go]: error string to lower case

**Comment by aboch** [comment]
LGTM

**Comment by mrjana** [comment]
LGTM

**Pull request merged**